src/App.js

```diff
@@ -3,6 +3,15 @@ import './App.css';
 
 import todos from './api/todos';
 import users from './api/users';
+import TodoList from './components/TodoList';
+
+const preparedTodos = todos.map((item) => {
+  const tempObj = { ...item };
+
+  tempObj.user = users.find(user => (user.id === tempObj.userId));
+
+  return tempObj;
+});
```

Comment:
```suggestion
const preparedTodos = todos.map((item) => ({
  ...item,
  user: users.find(user => user.id === item.userId),
}));
```
Используй деструктуризацию